### PR TITLE
Update 2020-05-04-dvc-3-years-and-1-0-release.md

### PR DESCRIPTION
**Pull request by efiop**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please chose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by efiop** [comment]
Please do not merge, testing things out...

**Comment by efiop** [comment]
Ok, pre release is finally out and works https://pypi.org/project/dvc/#history

File: content/blog/2020-05-04-dvc-3-years-and-1-0-release.md [hunk 1]
```diff
@@ -86,12 +86,16 @@ is growing every week. Our [Discord channel](http://dvc.org/chat) has almost two
...
 everyone willing to try out DVC, thank you for the opportunity.
 
-## DVC 1.0. is the result of 3 years of learning
+## DVC 1.0 is the result of 3 years of learning
 
 All these contributions, big and small, have a collective impact on DVC's
 development. I'm happy (and a bit nervous) to announce that a pre-release of a
 brand new DVC 1.0 is ready for public beta testing.
 
+```dvc
```
Comment (dmpetrov) [inline content/blog/2020-05-04-dvc-3-years-and-1-0-release.md:95]: You can add: 
`The installation instruction is on [our website](https://dvc.org/doc/install/pre-release) or you can install from pip:`

Comment (dmpetrov) [reply content/blog/2020-05-04-dvc-3-years-and-1-0-release.md:95]:
````
You can install the pre-release version from the development branch (instruction on [our website](https://dvc.org/doc/install/pre-release)) or you can from pip:
```

Comment (efiop) [reply content/blog/2020-05-04-dvc-3-years-and-1-0-release.md:95]: Fixed. Accidentally commited directly without creating a PR :(

**Pull request merged**